### PR TITLE
Add-onnx-workaround-benchmark-to-corresponding-bundles

### LINKED ISSUE #346: Onnx workaround for TensorRT export
**Is your feature request related to a problem? Please describe.**
When converting through `torch.jit.script`, a subset models in the model zoo suffer the slowdown issue or encounter the fail conversion. Fortunatelly, there is an onnx-tensorrt workaround for these models.

**Describe the solution you'd like**
Add the onnx workaround command for the conversion.
Benchmark models, who suffer the slowdown issue through `torch.jit.script`, on onnx workaround.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by binliunls**
Fixes #346  .

### Description
Add the ONNX-TensorRT conversion command to the `CONTRIBUTING.md`.
Add the ONNX-TensorRT benchmark results to the bundles using this conversion.

### Status
Work in progress

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -130,9 +130,11 @@ Check the functionality of exporting the checkpoint to [TensorRT](https://develo
 python -m monai.bundle trt_export --net_id network_def --filepath models/model_trt.ts --ckpt_file models/model.pt --meta_file configs/metadata.json --config_file configs/inference.json --precision <fp32/fp16> --dynamic_batchsize "[min, opt, max]"
 ```
 
-The other way to export your pytorch model to a TensorRT engine based torchscript is：
-1. Export the model to a **TensorRT engine** through [onnx](https://pytorch.org/docs/stable/onnx.html).
-1. Use the `torch_tensorrt.ts.embed_engine_in_new_module` API in [this link](https://pytorch.org/TensorRT/py_api/ts.html) to wrap the **TensorRT engine** to a **TensorRT engine based torchscript**.
+The other way to export a pytorch model to a TensorRT engine-based TorchScript is through the ONNX-TensorRT. This way can solve the slowdown issue of some models. Please notice that by defaulf the ONNX-TensorRT way assign only one output to the model. If a model has many outputs, please use the `--output_names` to specify the name of outputs.
```
Comment (yiheng-wang-nv) [inline CONTRIBUTING.md]: small typo:
> the ONNX-TensorRT way **assign** only one output to the model

assigns

File: models/brats_mri_segmentation/docs/README.md [hunk 2]
```diff
@@ -64,6 +64,23 @@ Please refer to https://pytorch.org/docs/stable/notes/randomness.html#reproducib
 #### Validation Dice
 ![A graph showing the validation mean dice over 300 epochs](https://developer.download.nvidia.com/assets/Clara/Images/monai_brats_mri_segmentation_val.png)
 
+#### TensorRT speedup
+The `endoscopic_inbody_classification` bundle supports the TensorRT acceleration through the ONNX-TensorRT way. The table below shows the speedup ratios benchmarked on an A100 80G GPU, in which the `model computation` means the speedup ratio of model's inference with a random input without preprocessing and postprocessing and the `end2end` means run the bundle end to end with the TensorRT based model. The `torch_fp32` and `torch_amp` is for the pytorch model with or without `amp` mode. The `trt_fp32` and `trt_fp16` is for the TensorRT based model converted in corresponding precision. The `speedup amp`, `speedup fp32` and `speedup fp16` is the speedup ratio of corresponding models versus the pytorch float32 model, while the `amp vs fp16` is between the pytorch amp model and the TensorRT float16 based model. Currently, this model can only be accelerated through the ONNX-TensorRT way and the Torch-TensorRT way will be comming soon.
```
Comment (yiheng-wang-nv) [inline models/brats_mri_segmentation/docs/README.md]: `endoscopic_inbody_classification`  should be `brats_mri_segmentation`

Comment (yiheng-wang-nv) [reply models/brats_mri_segmentation/docs/README.md]: typo: `comming`, suggest to use `will come soon`

File: models/endoscopic_inbody_classification/docs/README.md [hunk 3]
```diff
@@ -67,6 +67,23 @@ Accuracy was used for evaluating the performance of the model. This model achiev
 #### Validation Accuracy
 ![A graph showing the validation accuracy over 25 epochs.](https://developer.download.nvidia.com/assets/Clara/Images/monai_endoscopic_inbody_classification_val_accuracy.png)
 
+#### TensorRT speedup
+The `endoscopic_inbody_classification` bundle supports the TensorRT acceleration through the ONNX-TensorRT way. The table below shows the speedup ratios benchmarked on an A100 80G GPU, in which the `model computation` means the speedup ratio of model's inference with a random input without preprocessing and postprocessing and the `end2end` means run the bundle end to end with the TensorRT based model. The `torch_fp32` and `torch_amp` is for the pytorch model with or without `amp` mode. The `trt_fp32` and `trt_fp16` is for the TensorRT based model converted in corresponding precision. The `speedup amp`, `speedup fp32` and `speedup fp16` is the speedup ratio of corresponding models versus the pytorch float32 model, while the `amp vs fp16` is between the pytorch amp model and the TensorRT float16 based model. Currently, this model can only be accelerated through the ONNX-TensorRT way and the Torch-TensorRT way will be comming soon.
```
Comment (yiheng-wang-nv) [inline models/endoscopic_inbody_classification/docs/README.md]: `comming` same comment as above

File: models/brats_mri_segmentation/docs/README.md (hunk 2 above)
Comment (yiheng-wang-nv) [reply models/brats_mri_segmentation/docs/README.md]: > The table below shows the speedup ratios benchmarked on an A100 80G GPU, in which the `model computation` means the speedup ratio of model's inference with a random input without preprocessing and postprocessing and the `end2end` means run the bundle end to end with the TensorRT based model. The `torch_fp32` and `torch_amp` is for the pytorch model with or without `amp` mode ...

This sentence is too long, suggest to rephrase like:

The table below shows the speedup ratios benchmarked on an A100 80G GPU.

| method | torch_fp32(ms) | torch_amp(ms) | trt_fp32(ms) | trt_fp16(ms) | speedup amp | speedup fp32 | speedup fp16 | amp vs fp16|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| model computation | 5.49 | 4.36 | 2.35 | 2.09 | 1.26 | 2.34 | 2.63 | 2.09 |
| end2end | 592.01 | 434.59 | 395.73 | 394.93 | 1.36 | 1.50 | 1.50 | 1.10 |

Where:
- `model computation` means the speedup ratio of model's inference with a random input without preprocessing and postprocessing.
- `end2end` means run the bundle end to end with the TensorRT based model.
- The `torch_fp32` and `torch_amp` are for the pytorch model with or without `amp` mode.
...

What do you think? May also need to modify 3 TRT bundles changed before.
May also need @zephyrie 's help to review the content to ensure the doc quality.

**Comment by yiheng-wang-nv** [comment]
Thanks @binliunls , I'm fine with the latest changes. I will add the CI test for this changes via push a commit to this PR directly.

**Review by yiheng-wang-nv — APPROVED** [review]
